bricksrc/properties.py

```diff
@@ -146,4 +146,10 @@
         RDFS.domain: BRICK.Tag,
         RDFS.range: OWL.Class,
     },
+    "hasAddress": {
+        RDFS.subPropertyOf: VCARD.hasAddress,
+        SKOS.definition: Literal("To specify the address of a location."),
+        RDFS.domain: BRICK.Location,
```

Comment:
Do we want the domain of `hasAddress` to be a `brick:Building`? I guess if we agree that multiple Brick locations (e.g. it might make sense to have `hasAddress` apply to `brick:Site` as well), then we need to use `brick:Location` instead of a more specific class. I'm open either way, though I do like the idea of saying that an address *must* be associated with a Building

Reply:
Addresses can be in different granularities, I think. For example, a room can have an address. Unless we can have an exhaustive list of location types that can be associated with an address, I would like to allow any location to have an address. Let me know what you think.

Reply:
According to [the vCard documentation](https://www.w3.org/TR/vcard-rdf/#Delivery_Addressing_Properties) it looks like an address refers to a street address. I'm ok with opening up the idea of an address to more than just buildings, but this would require expanding the definition of address beyond what is provided by the vCard ontology

Reply:
Good catch. I just looked up a bit more and there was a corresponding concept for that: [extended address](https://www.w3.org/TR/vcard-rdf/#d4e860), as described here: https://tools.ietf.org/id/draft-ietf-vcarddav-vcardrev-02.html#ADR
The docs do not explain why it has been deprecated. If this is the case, I agree with your suggestion. I will change the domain of hasAddress to Building only.

Reply:
I'll keep an eye out for an extended address format/standard that we might be able to apply. Might be an interesting place to use a building/geographic coordinate system? Maybe there are standards for indoor localization or mapping?